airflow/jobs/__init__.py

```diff
@@ -16,10 +16,3 @@
 # specific language governing permissions and limitations
 # under the License.
 #
-
-# pylint: disable=missing-docstring
-
-from airflow.jobs.backfill_job import BackfillJob  # noqa: F401
-from airflow.jobs.base_job import BaseJob  # noqa: F401
-from airflow.jobs.local_task_job import LocalTaskJob  # noqa: F401
-from airflow.jobs.scheduler_job import SchedulerJob  # noqa: F401
```

Comment:
Does this requires note in `UPDATING.md`?

Reply:
Doubt it. It's really an internal stuff. I see no use of it in Dags (and I think this should be our trigger for UPDATING.md - if you need to change Airlfow configuration or modify your dags or you have different cli/web behaviour.